Signals: Quandl example script (#68)
numerai/example-scripts: signals/quandl/README.md

> This script uses Quandl's [End of Day US Stock Prices](https://www.quandl.com/data/EOD-End-of-Day-US-Stock-Prices/) data. (It's a premium data source)
> 
> Two ways to download the data
> 
> 1. Load every ticker and its daily features sequentially. (slower)
> 2. Load entire dataset once and load data from it. (much faster)
> 
> This script provides functions for both ways of loading the data in same format.

Reviewer: ```suggestion
This script uses Quandl's [end of day US stock prices](https://www.quandl.com/data/EOD-End-of-Day-US-Stock-Prices/) data. 
```

> *Need to install quandl's python package and numerapi*

Reviewer: ```suggestion
*Before running, install quandl's python package and numerapi*
```

```
pip install quandl
pip install numerapi
```

> **`API_KEY = "<Quandl API KEY>"` This need to be set before running the script.**

Reviewer: ```suggestion
**`API_KEY = "<Quandl API KEY>"` This needs to be set before running the script.**

To get an API key, create a paid account on Quandl.
```
